ui/src/app/utils/workbench-gapi-client.ts

```diff
@@ -23,15 +23,17 @@ export async function getBillingAccountInfo(googleProject: string) {
             .getBillingInfo({
               name: 'projects/' + googleProject,
             })
-            .then((response) => resolve(JSON.parse(response.body)));
+            .then((response) => resolve(JSON.parse(response.body)))
+            .catch((error) => reject(error));
         });
       } else {
         gapi.client.load('cloudbilling', 'v1', () => {
           gapi.client.cloudbilling.projects
             .getBillingInfo({
               name: 'projects/' + googleProject,
             })
-            .then((response) => resolve(JSON.parse(response.body)));
+            .then((response) => resolve(JSON.parse(response.body)))
+            .catch((error) => reject(error));
```

Comment:
this is the line that fixed it for me.  The exception was not able to be handled because this was skipped